src/pr-gating/CIVerify.yml

```diff
@@ -1,42 +1,45 @@
-# This is a starter pipeline to handle Gated PRs using GHAzDO. Currently (October 2023) this is not supported out of the box by the product, 
-# it probably will be supported in the future, this project can be used as a workaround until that time. 
-# We want to restrict new code going into main and only allow PRs if the new code does not introduce any new CodeQL issues. 
-# The idea is to set a branch protection policy (for main), forcing this pipeline to succeed before a PR into main can happen. 
-# The pipeline will run CodeQL on the source branch of the PR. Later, using a PowerShell script, the CodeQL issues of the PR source and target will be compared. 
-# If there are issues in the PR source that are not in main, this pipeline will fail. 
+# This is a starter pipeline to handle Gated PRs using GHAzDO. Currently (October 2023) this is not supported out of the box by the product,
+# it probably will be supported in the future, this project can be used as a workaround until that time.
+# We want to restrict new code going into main and only allow PRs if the new code does not introduce any new CodeQL or Dependency Scanning issues.
+# The idea is to set a branch protection policy (for main), forcing this pipeline to succeed before a PR into main can happen.
+# The pipeline will run CodeQL and Dependency Scanning on the source branch of the PR. Later, using a PowerShell script, the security alerts of the PR source and target will be compared.
+# If there are alerts in the PR source that are not in main, this pipeline will fail.
 #
-# If new alerts are detected these needs to be analysed using the regular Advanced Security Code Scanning UI. Set the filter to the pr branch and fix or dissmiss all the new issue. 
-# After that, the PR check CIVerify can be requeued in the PR. Hopefully this time, without any issues. 
-# 
-# The script needs a PAT to run (for accessing the REST API). This PAT should be setup as a secret variable for the pipleline (name: GATING_PAT). 
+# If new alerts are detected these needs to be analysed using the regular Advanced Security Code Scanning UI.
+#  - For Code Scanning, use the filter to the pr branch and fix or dissmiss all the new issues.
```

Comment:
Yup that is why I split this out and called it out separately from the Code Scanning alerts.  This may change in the future , but so will most of this project once PR integration is handled inside the product.